polars/polars-lazy/polars-plan/src/dsl/mod.rs

```diff
@@ -1994,6 +1994,22 @@ impl Expr {
         .with_fmt("log")
     }
 
+    #[cfg(feature = "log")]
+    /// Compute the natural logarithm of all elements plus one in the input array
+    pub fn log1p(self) -> Self {
```

Comment:
We are in the process of re-writing the direct `map/apply` functions to `map_private` so that they can be serialized. Could you rewrite this in such a way? See `rolling_skew` for what I mean.

Reply:
Lmk if it looks good now (I moved the `entropy` method to reduce the number of `#[cfg]` blocks). If yes, happy to make the transition for the other log-related functions in a separate PR.

Reply:
Yes, this looks good! If you'd be willing to move the other ones as well, that'd be great.

Reply:
Realized that I was accidentally using `apply_private` instead of `map_private`; fixed that now.